plaidml/edsl/tests/edsl_test.cc

```diff
@@ -2093,5 +2093,209 @@ TEST_F(CppEdsl, LayerException) {
   runProgram(program);
 }
 
+TEST_F(CppEdsl, ArgSort1d) {
+  auto I = Placeholder(DType::FLOAT32, {20});
+  auto O = argsort(I, /*axis=*/0);
+  auto program = makeProgram("argsort", {I}, {O});
+  std::vector<float> input = {
```

Comment:
Can you put a lit test in here right after makeProgram? I would guess you could nearly copy one of the conversion lit tests to accomplish this (but I might be overlooking something)